lib/wit.js

```diff
@@ -29,8 +29,8 @@ function Wit(opts) {
     const handler = makeWitResponseHandler(logger, 'message');
     logger.debug(method, fullURL);
     return fetch(fullURL, {
-      method,
-      headers,
+      method:method,
```

Comment:
this should not make a difference -- the code to the left is simply object short syntax